### PR TITLE
[ErrorHandler] Add types to private properties

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/ErrorHandler/ErrorHandler.php [hunk 1]
```diff
@@ -411,10 +410,7 @@ public function handleError(int $type, string $message, string $file, int $line)
 
         $logMessage = $this->levels[$type].': '.$message;
 
-        if (null !== self::$toStringException) {
```
Comment (derrabus) [inline src/Symfony/Component/ErrorHandler/ErrorHandler.php:414]: Maybe I'm missing something, but I haven't found a reason why this variable should have any other value than `null`.

Comment (nicolas-grekas) [reply src/Symfony/Component/ErrorHandler/ErrorHandler.php:414]: that's a leftover, it can be cleaned up

Comment (derrabus) [reply src/Symfony/Component/ErrorHandler/ErrorHandler.php:414]: Indeed. The variable was used as a workaround on PHP 7.2/7.3 and became obsolete when we removed compat code on 6.0.

**Comment by nicolas-grekas** [comment]
Thank you @derrabus.

**Pull request merged**